typhos/display.py

```diff
@@ -1405,6 +1405,9 @@ def add_device(self, device, macros=None):
         self.macros = self._build_macros_from_device(device, macros=macros)
         self.load_best_template()
 
+        if not self.windowTitle():
+            self.setWindowTitle(getattr(device, "name", ""))
```

Comment:
Out of curiosity: which edge case is this fixing?

Reply:
All widgets in Qt can have window titles, as you already know, in case they're displayed as-is in the window manager.

`TyphosDeviceDisplay` or its subclasses may choose to customize the window title but they don't currently. This says "if there's no title set, at least give it the device name"

Then to answer your actual question: this title is then used during the screenshot process, with output saying "Taking a screenshot of (suite title) (device display title)":
`[2023-08-09 09:40:41] - INFO - Saving screenshot of 'Typhos Suite - EpicsMotor': 'EpicsMotor' to 'EpicsMotor.png'`